Integrations/integration-Cymon.yml

```diff
@@ -0,0 +1,543 @@
+                'Tags': reports.get(report, {}).get('tags'),
+                'Hostname': reports.get(report, {}).get('ioc', {}).get('hostname'),
+                'IP': reports.get(report, {}).get('ioc', {}).get('ip', {}),
+                'Domain': reports.get(report, {}).get('ioc', {}).get('domain'),
+                'Reported By': reports.get(report, {}).get('reported_by'),
+                'Location': reports.get(report, {}).get('location')
+            })
+
+        return {
+            'reports': report_results,
+            'total': total
+            }
+
+    def create_domain_command_markdown(domain, total_hits, reports, domain_full_report):
+        md = '## Cymon Domain report for: {}\n'.format(domain)
+
+        md += '\n'
+
+        md += '**Total Hits:** {}'.format(total_hits)
+
+        md += '\n'
+
+        md += tableToMarkdown("The following reports are the latest malicious hits resolved to the given domain:", reports, ['Title', 'Hostname', 'IP', 'Timestamp', 'Feed', 'Tags', 'Location', 'Reported By', 'Domain'])
+
+        if len(domain_full_report) > 0:
+           md += tableToMarkdown("Full report list:", domain_full_report, ['Title', 'Hostname', 'IP', 'Timestamp', 'Feed', 'Tags', 'Location', 'Reported By', 'Domain'])
+
+        return md
+
+    def create_context_domain_command(domain, reports):
+
+        cymon_domain_context_activities = []
+        description = 'Reported suspicious activities: '
+
+        for report in reports:
+            cymon_domain_context_activities.append({
+                'Title': report.get('Title'),
+                'Tags': report.get('Tags'),
+                'Time': report.get('Timestamp'),
+                'Hostname': report.get('Hostname'),
+                'IP': report.get('IP')
+            })
+
+            description += '{}, '.format(report.get('Title'))
+
+        description = description[:-2]
+
+        context = {
+            outputPaths['domain']: {
+                'Name': domain,
+                'Malicious': {
+                    'Vendor': 'Cymon',
+                    'Description': description
+                }
+            },
+            'Cymon': {
+                'Domain': {
+                    'Activities': cymon_domain_context_activities
+                }
+            }
+        }
+
+        return context
+
+    def get_domain_report_command():
+        args = demisto.args()
+
+        domain = args.get('domain')
+        full_response = args.get('fullResponse') == 'true'
+
+        domain_report = get_domain_report(domain)
+        domain_full_report = []
+
+        if full_response:
+            domain_full_report = get_domain_full_report(domain)
+
+        if not domain_report.get('reports') or not domain_report.get('total'):
+            return "Domain " + domain + " is not in Cymons's dataset"
+
+        mark_down = create_domain_command_markdown(domain, domain_report.get('total'), domain_report.get('reports'), domain_full_report)
```

Comment:
@MarkShleifer variable name should be `markdown`

Reply:
Done